backend/src/openarchiefbeheer/destruction/api/mixins.py

```diff
@@ -0,0 +1,26 @@
+from django.utils.translation import gettext_lazy as _
+
+from rest_framework.exceptions import ValidationError
+from rest_framework.request import Request
+
+from ..models import DestructionList
+
+
+class DestructionListChecksMixin:
+    def get_destruction_list(self, request: Request) -> DestructionList:
+        try:
```

Comment:
This feels like a public api, but using it that way (withou calling `check_destruction_list_permissions()` would bypass permission checks. Would it be nice to somehow specify the public api, and do all the required checks here, instead of assuming that the implementation calls it manually?